comment/api/permissions.py

```diff
@@ -22,6 +24,25 @@ def has_object_permission(self, request, view, obj):
         return obj.user == request.user
 
 
+class UserPermittedOrReadOnly(permissions.BasePermission):
+    message = BlockedUserMSG.NOT_PERMITTED
+
+    def has_permission(self, request, view):
+        return bool(
+            request.method in permissions.SAFE_METHODS or
+            not settings.COMMENT_ALLOW_BLOCKING_USERS
+            or not BlockedUser.objects.is_user_blocked(request.user.id, request.POST.get('email'))
```

Comment:
shouldn't this be a separate permission? maybe you can add this as a separate private method to `CanCreatePermission`.

Reply:
`UserPermittedOrReadOnly` is a separate permission and it's not only used with create comment, but with all other actions.

Reply:
okay but i was asking you to remove the code to check if the user is blocked or not. from what i understand, blocking thing will should be used to create comment. i think blocking them to reactions etc maybe implemented at a later stage if required.